addons/website_event_track_quiz/static/src/scss/event_quiz.scss

```diff
@@ -0,0 +1,155 @@
+// Retrive the tab's height by summ its properties
```

Comment:
Mmmh, so much required css ? Maybe we will try to clean with v2 (+ remove slides references :D )